README.md

```diff
@@ -0,0 +1,684 @@
+|---------|-------------|----------|----------------------------------------------|
+| `ucd`   | `&Payload`  | Yes      | The CID of the [Delegation Payload][Payload] |
+| `sig`   | `Signature` | Yes      | The `iss`'s [Signature] over the `ucd` field |
+
+
+# 3. Delegation Payload
+
+The payload MUST describe the authorization claims, who is involved, and its validity period.
+
+| Field | Type                                      | Required | Description                                                 |
+|-------|-------------------------------------------|----------|-------------------------------------------------------------|
+| `udv` | `String`                                  | Yes      | UCAN Semantic Version (`1.0.0-rc.1`)                        |
+| `iss` | `DID`                                     | Yes      | Issuer DID (sender)                                         |
+| `aud` | `DID`                                     | Yes      | Audience DID (receiver)                                     |
+| `nbf` | `Integer` (53-bits[^js-num-size])         | No       | "Not before" UTC Unix Timestamp in seconds (valid from)     |
+| `exp` | `Integer \| null` (53-bits[^js-num-size]) | Yes      | Expiration UTC Unix Timestamp in seconds (valid until)      |
+| `nnc` | `String`                                  | Yes      | Nonce                                                       |
+| `mta` | `{String : Any}`                          | No       | [Meta] (asserted, signed data) — is not delegated authority |
+| `sub` | `DID`                                     | Yes      | Principal that the chain is about (the [Subject])           |
+| `can` | `String`                                  | Yes      | [Ability]                                                   |
+| `iff` | `[Caveat]`                                | Yes      | [Caveat]s                                                   |
+
+The payload MUST be serialized as [IPLD] and [signed over][Envelope]. The RECOMMENDED IPLD codec is [DAG-CBOR].
+
+## 3.1 Version
+
+The `udv` field sets the version of the UCAN Delegation specification used in the payload.
+
+## 3.2 Principals
+
+The `iss` and `aud` fields describe the token's principals. They are distinguished by having DIDs. These can be conceptualized as the sender and receiver of a postal letter. The token MUST be signed with the private key associated with the DID in the `iss` field. Implementations MUST include the [`did:key`] method, and MAY be augmented with [additional DID methods][DID].
+
+The `iss` and `aud` fields MUST contain a single principal each.
+
+If an issuer's DID has multiple or mutable keys (e.g. [`did:plc`], [`did:web`]), the key used to sign the UCAN MUST be made explicit, using the [DID fragment] (the hash index) in the `iss` field. The `aud` field SHOULD NOT include a hash field, as this defeats the purpose of delegating to an identifier for multiple keys instead of an identity.
```

Comment:
haha I like how we've changed positions on this since a much older version. I'm okay with that, sure. Will update 👍

Reply:
> The `aud` field SHOULD NOT include a hash field, as this defeats the purpose of delegating to an identifier for multiple keys instead of an identity.

> include a hash field

what does 'a hash field' mean here? (i.e. how would i implement/test this behavior)